$NetBSD: patch-src_CMakeLists.txt,v 1.2 2025/02/07 14:24:42 wiz Exp $

CMake Error at src/CMakeLists.txt:83 (if):
  if given arguments:

    "ENABLE_GDBINDEX" "AND" "NetBSD" "MATCHES" "Linux" "AND" "STREQUAL" "Debug"

  Unknown arguments specified

https://invent.kde.org/sdk/kdiff3/-/issues/51

--- src/CMakeLists.txt.orig	2025-02-07 13:55:19.772490605 +0000
+++ src/CMakeLists.txt
@@ -80,7 +80,7 @@ if(APPLE)
   set(MACOSX_BUNDLE_COPYRIGHT "2003- The KDiff3 Authors")
 endif()
 
-if(ENABLE_GDBINDEX AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND ${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+if(ENABLE_GDBINDEX AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
     find_program(GDBINDEX "gdb-add-index")
     if(GDBINDEX)
         add_custom_command(TARGET kdiff3 POST_BUILD
